README.md

```diff
@@ -15,6 +15,31 @@ This would create a project directory \<project> ("projrouter" in case of the ab
 
         vivado_hls -p <project>
 
+### Track Quality Specific Instructions
```

Comment:
Could you add instructions for producing hls_features.dat and hls_hex.dat? If there are instructions elsewhere, you could just link them here.

Reply:
I see. Maybe you could put this explanation in the README with a link to your CMSSW branch? Then we can update this whenever the emulation changes are all merged.